src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -2588,6 +2588,12 @@ void ViewProviderSketch::updateColor()
     editCoinManager->updateColor();
 }
 
+bool ViewProviderSketch::selectAll()
+{
+    // TODO: eventually implement "select all" logic
```

Comment:
Why leaving this as a TODO? It seems rather straight forward. Unless you want to split in 2 PR